eventkit_cloud/ui/static/ui/app/components/DashboardPage/DashboardPage.js

```diff
@@ -0,0 +1,521 @@
+    }
+
+    refreshRecentlyViewed({ isAuto = false } = {}) {
+        this.props.getViewedJobs({
+            pageSize: this.getGridColumns({ getMax: true }) * 3,
+            isAuto,
+        });
+    }
+
+    refresh({ isAuto = false } = {}) {
+        this.refreshMyDataPacks({ isAuto });
+        this.refreshFeatured({ isAuto });
+        this.refreshRecentlyViewed({ isAuto });
+    }
+
+    autoRefresh() {
+        this.refresh({ isAuto: true });
+    }
+
+    handleNotificationsViewAll() {
+        browserHistory.push('/notifications');
+    }
+
+    handleFeaturedViewAll() {
+        browserHistory.push('/exports');
+    }
+
+    handleMyDataPacksViewAll() {
+        browserHistory.push('/exports?collection=myDataPacks');
+    }
+
+    handleShareOpen(run) {
+        this.setState({ shareOpen: true, targetRun: run });
+    }
+
+    handleShareClose() {
+        this.setState({ shareOpen: false, targetRun: null });
+    }
+
+    handleShareSave(perms) {
+        this.handleShareClose();
+        const permissions = { ...perms };
+        this.props.updateDataCartPermissions(this.state.targetRun.job.uid, permissions);
+    }
+
+    isLoading() {
+        return (
+            this.state.loadingPage ||
+            this.props.runsDeletion.deleting ||
+            this.props.updatePermission.updating
+        );
+    }
+
+    render() {
+        const mainAppBarHeight = 95;
+        const pageAppBarHeight = 35;
+        let styles = {
+            root: {
+                position: 'relative',
+                height: window.innerHeight - mainAppBarHeight,
+                width: '100%',
+                backgroundImage: `url(${backgroundUrl})`,
+                color: 'white',
+            },
+            appBar: {
+                backgroundColor: '#161e2e',
+                height: '35px',
+                color: 'white',
+                fontSize: '14px',
+                zIndex: '0',
+            },
+            customScrollbar: {
+                height: window.innerHeight - mainAppBarHeight - pageAppBarHeight,
+            },
+            pageTitle: {
+                fontSize: '18px',
+                lineHeight: '35px',
+                paddingLeft: '10px',
+                height: '35px',
+            },
+            loadingOverlay: {
+                position: 'absolute',
+                height: '100%',
+                width: '100%',
+                background: 'rgba(0,0,0,0.5)',
+                zIndex: '100',
+            },
+            loadingPage: {
+                position: 'absolute',
+                left: '50%',
+                top: '50%',
+                transform: 'translate(-50%, -50%)',
+            },
+            content: {
+                marginBottom: '12px',
+                maxWidth: '1920px',
+                margin: 'auto',
+            },
+            noData: {
+                margin: `0 ${10 + this.getGridPadding()/2}px`,
+                padding: '29px',
+                fontSize: '25px',
```

Comment:
This could just be personal preference, but do you think this text size is awkwardly large? I was kinda taken aback by the massive text when I first loaded the screen.  (this comment also applies to some other spots in notifications/dashboard page)
![large_text](https://user-images.githubusercontent.com/16799449/39720930-db8f279e-520b-11e8-88fc-83265dde8b14.PNG)

Reply:
Haha it is a bit chunky for sure. I tried to match Liz's designs as closely as possible, so this might be something to take up with her. I think it could be a little smaller for sure.

Reply:
I think for consistency throughout the application it should be okay to go ahead and make the text smaller. This would end up being the only page with larger text if not changed.

Reply:
Cool, I'll go ahead and shrink it down a little bit.

Reply:
Btw, I'll make sure to shrink the text in NotificationsPage as well.

Reply:
Fixed as of dfb5cf9cecde835e415bff83c46a86b154bdad1b